src/network/mcpe/StandardEntityEventBroadcaster.php

```diff
@@ -67,7 +67,7 @@ public function syncAttributes(array $recipients, Living $entity, array $attribu
 		if(count($attributes) > 0){
 			$this->sendDataPacket($recipients, UpdateAttributesPacket::create(
 				$entity->getId(),
-				array_map(fn(Attribute $attr) => new NetworkAttribute($attr->getId(), $attr->getMinValue(), $attr->getMaxValue(), $attr->getValue(), $attr->getDefaultValue(), []), $attributes),
+				array_map(fn(Attribute $attr) => new UpdateAttribute($attr->getId(), $attr->getMinValue(), $attr->getMaxValue(), $attr->getValue(), $attr->getMinValue(), $attr->getMaxValue(), $attr->getDefaultValue(), []), $attributes),
```

Comment:
Why are the min/max repeated?

Reply:
On the protocol side there are some new fields: `float $defaultMin, float $defaultMax`, don't know what those for, maybe dries could explain

Reply:
I see. Probably because the min/max can be altered by modifiers.